miniscope_io/stream_daq.py

```diff
@@ -455,8 +457,11 @@ def _format_frame(
         try:
             for frame_data, header_list in exact_iter(frame_buffer_queue.get, None):
 
-                locallogs.debug("Found frame in queue")
+                if not frame_data:
```

Comment:
I *think* this should be fine, but at the risk of repeating myself a million times it would be really nice to have tests here because the termination logic is a little screwy - where passing `None` propagates through the processes to stop them all when their queues are finished, but still, fine for this PR, i'll work on breaking it up after

Reply:
I would avoid doing that before an announced big refactor, which I'm not aware of how it'll look, but sure, I've heard that a million times and understand. Anyway, we need to look at the termination logic because this code shouldn't stop even if there's a few-second break in the data stream. So when running with actual devices, it can just be `ctrl-c`. I think we only need this `None` propagation termination for testing or replay mode.